scripts/utils.sh

```diff
@@ -123,4 +127,16 @@ function setTraefikHostnames {
     export KEYCLOAK_HOSTNAME=auth-"${IP_WITH_DASHES}.traefik.me"
     echo "→ SUPERSET_HOSTNAME=$SUPERSET_HOSTNAME"
     echo "→ KEYCLOAK_HOSTNAME=$KEYCLOAK_HOSTNAME"
+}
+
+function setDockerComposeCLIOptions () {
+    # Parse 'docker-compose-files.txt' to get the list of Docker Compose files to run
+    dockerComposeFiles=$(cat docker-compose-files.txt)
+    for file in ${dockerComposeFiles}
+    do
+        export dockerComposeCLIOptions="$dockerComposeCLIOptions -f ../docker/$file"
+    done
+    if [ "$withOzoneSSO" = "true" ]; then
```

Comment:
Where is `withOzoneSSO` set?

Reply:
We need to provide `withOzoneSSO` and set it to true if we want to enable the sync. By default, it is not started.